cli/foundation-models/system/inference/text-to-image/text-to-image-online-endpoint.sh

```diff
@@ -49,12 +49,18 @@ az ml online-endpoint create --name $endpoint_name $workspace_info  || {
 }
 
 # Deploy model from registry to endpoint in workspace
-az ml online-deployment create --file deploy-online.yaml $workspace_info --all-traffic --set \
-  endpoint_name=$endpoint_name model=azureml://registries/$registry_name/models/$model_name/versions/$model_version \
-  instance_type=$deployment_sku || {
-    echo "deployment create failed"; exit 1;
+az ml online-deployment create --file deploy-online.yaml $workspace_info --set \
+endpoint_name=$endpoint_name model=azureml://registries/$registry_name/models/$model_name/versions/$model_version \
+instance_type=$deployment_sku || {
+echo "deployment create failed"; exit 1;
 }
+yaml_file="deploy-online.yaml"
+get_yaml_value() {
+    grep "$1:" "$yaml_file" | awk '{print $2}' | sed 's/[",]//g'
+}
+traffic=$(get_yaml_value "name")
```

Comment:
change this "traffic" to deployment-name

Reply:
ok